rmDevice.py

```diff
@@ -9,15 +10,14 @@
 parser = ArgumentParser()
 parser.add_argument("username")
 args = parser.parse_args()
-# parser.add_argument("Roles", nargs='+')
 
-session = sessionmaker(bind=create_engine('sqlite:///HardwareCheckout/db.sqlite'))
+session = sessionmaker(bind=create_engine(db_path))
```

Comment:
Wow! Thanks for this! I didn't realize that this file never got updated!

Reply:
👍